tests/unit/test_simulated_stack.py

```diff
@@ -155,3 +159,29 @@ def testCallSetsSearch(self):
             self.assertEqual(callSetName, callSet.sampleId)
 
         # TODO add tests after name string search schemas is implemented
+
+    def testReads(self):
+        # search read group sets
+        path = utils.applyVersion('/readgroupsets/search')
+        request = protocol.SearchReadGroupSetsRequest()
+        request.datasetIds = ['simulatedDataset1']
+        response = self.sendJsonPostRequest(path, request.toJsonString())
+        self.assertEqual(response.status_code, 200)
+        responseData = protocol.SearchReadGroupSetsResponse.fromJsonString(
+            response.data)
+        readGroupSets = responseData.readGroupSets
+        self.assertEqual(len(readGroupSets), 1)
+
+        # search reads
+        path = utils.applyVersion('/reads/search')
+        request = protocol.SearchReadsRequest()
+        readGroupId = 'aReadGroupSet:one'
```

Comment:
This is pretty brittle --- why not get the readGroupId from the readGroupSets object you have above?